src/index.html

```diff
@@ -15,7 +16,7 @@ <h1 class="main-title">Leaderboard</h1>
 		<section>
 			<div class="sub-title">
 				<h2>Recent score</h2>
-				<button type="button" class="refresh">Refresh</button>
+				<button type="button" class="refresh"><i class="fa fa-refresh" id="icon"></i></button>
 			</div>
 			<div class="list-players"></div>
 		</section>
```

Comment:
- [ ] [OPTIONAL] Kindly sort the scores, either alphabetically or numerically. This will ensure a better scoreboard or leaderboard viewing. check [this link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) to see the know how 😉.